ballista/rust/client/src/context.rs

```diff
@@ -198,14 +200,21 @@ impl BallistaContext {
                 DataFusionError::Internal("Received empty status message".to_owned())
             })?;
             let wait_future = tokio::time::sleep(Duration::from_millis(100));
+            let has_status_change = prev_status.map(|x| x != status).unwrap_or(false);
```

Comment:
```suggestion
            let has_status_change = prev_status.map(|x| x != status).unwrap_or(true);
```
I think the default should be `true` here. If there was no previous status then the status must have changed?

Reply:
You are right, sorry for the mistake :(